src/o365/outlook/commands/report/outlook-report-mailappusageversionsusercounts.ts

```diff
@@ -0,0 +1,38 @@
+import commands from '../../commands';
+import PeriodBasedReport from '../../../base/PeriodBasedReport';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+class OutlookMailAppUsageVersionsUserCountsCommand extends PeriodBasedReport {
```

Comment:
Following the naming convention, the class name should be `OutlookReportMailAppUsageVersionsUserCountsCommand`